sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -4377,6 +4377,11 @@ def test_gammas():
     assert xpretty(gamma(x), use_unicode=True) == u'Γ(x)'
 
 
+def test_deltas():
+    assert xpretty(DiracDelta(x), use_unicode=True) == u'δ(x)'
+    assert xpretty(DiracDelta(x, 1), use_unicode=True) == u'δ(x, 1)'
```

Comment:
IMHO ``DiracDelta(x - 1)`` should print as ``δ(x - 1)``. I like the way derivatives are printed in latex. See [this](https://gist.github.com/leosartaj/2267f0bb8fbd198de71312f9c3ffedb8).